src/js/medium-editor.js

```diff
@@ -1,4 +1,5 @@
 /*global module, console, define*/
+var DEBOUNCE_INTERVAL = 50;
```

Comment:
I don’t see any disadvantages from moving it to `debounce()` - it’s not used anywhere else and can be changed with the `wait` argument.